### PR TITLE
Refactor to use `isAtRule` util

### LINKED ISSUE #6291: Refactor to use `isAtRule` typeguard util
There many places in rules where instead of using existing util `isAtRule` we use `if (node.type === 'atrule')`. I suggest replace all `node.type === 'atrule'` with `isAtRule` because it makes code more readable and consistent.

### PR DESCRIPTION
**Pull request by fpetrakov**
> Which issue, if any, is this issue related to?

Closes #6291

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory

### CODE REVIEW

**Review by ybiquitous — APPROVED** [review]
@fpetrakov Thanks. LGTM 👍🏼

**Review by jeddy3 — APPROVED** [review]
Thank you!

File: lib/rules/string-quotes/index.js [hunk 1]
```diff
@@ -187,7 +188,7 @@ const rule = (primary, secondaryOptions, context) => {
 				return;
 			}
 
-			if (node.type === 'atrule' && node.name === 'charset') {
+			if (isAtRule(node) && node.name === 'charset') {
 				// allow @charset rules to have double quotes, in spite of the configuration
 				// TODO: @charset should always use double-quotes, see https://github.com/stylelint/stylelint/issues/2788
```
Comment (Mouvedia) [inline lib/rules/string-quotes/index.js:193]: seems like you can remove the "TODO: " prefix

Comment (ybiquitous) [reply lib/rules/string-quotes/index.js:193]: PR welcome!

**Pull request merged**